### PR TITLE
Implement Apache Airflow Task to Trigger Dag Runs

### PR DESCRIPTION
**Pull request by iNikitaGricenko**
### What changes are being made and why?

- Implemented Task that triggers dag run in Apache Airflow (#2)
---

### How the changes have been QAed?

```yaml
  - id: trigger_dag
    type: io.kestra.plugin.airflow.TriggerDagRun
    baseUrl: http://localhost:8080
    dagId: example_dag
    checkFrequency: PT30S
      interval: PT30S
      maxIterations: 100
      maxDuration: PT1H
    options:
      basicAuthUser: airflow
      basicAuthPassword: airflow
```

---

### Setup Instructions

- Apache Airflow docker container or local application. [Official documentation for installation in docker](https://airflow.apache.org/docs/apache-airflow/stable/howto/docker-compose/index.html)

### CODE REVIEW

File: src/main/java/io/kestra/plugin/airflow/dags/TriggerDagRun.java [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+    @Schema(
+        title = "The frequency at which to check the DAG run status"
+    )
+    @PluginProperty
+    @Builder.Default
+    private WaitFor.CheckFrequency checkFrequency = WaitFor.CheckFrequency.builder().build();
+
+    @Schema(
+        title = "Whether task should wait DAG run to complete",
```
Comment (loicmathieu) [inline src/main/java/io/kestra/plugin/airflow/dags/TriggerDagRun.java:108]:
```suggestion
        title = "Whether task should wait fr the DAG to run to completion",
```

File: src/main/java/io/kestra/plugin/airflow/dags/TriggerDagRun.java [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+        description = "Default value is false"
+    )
+    @PluginProperty
+    @Builder.Default
+    private Boolean wait = Boolean.FALSE;
+
+    @Schema(
+        title = "Overrides the default conf payload"
```
Comment (loicmathieu) [inline src/main/java/io/kestra/plugin/airflow/dags/TriggerDagRun.java:116]:
```suggestion
        title = "Overrides the default configuration payload"
```

File: src/main/java/io/kestra/plugin/airflow/dags/TriggerDagRun.java [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+    @NotNull
+    @PluginProperty(dynamic = true)
+    private String dagId;
+
+    @Schema(
+        title = "The frequency at which to check the DAG run status"
+    )
+    @PluginProperty
+    @Builder.Default
+    private WaitFor.CheckFrequency checkFrequency = WaitFor.CheckFrequency.builder().build();
```
Comment (loicmathieu) [inline src/main/java/io/kestra/plugin/airflow/dags/TriggerDagRun.java:105]: I don't think it's a good idea to use another task internal property class from the core plugin in another plugin.
I think here using only a timeout and ping the DAG endpoint each second would be enough it's what we did in a lot of other tasks.

File: src/test/java/io/kestra/plugin/airflow/dags/TriggerDagRunTest.java
Comment (loicmathieu) [inline src/test/java/io/kestra/plugin/airflow/dags/TriggerDagRunTest.java:1]: Is it possible to run Airflow easily with a simple docker compose so the test can be run inside our CI/CD

**Review by loicmathieu — CHANGES_REQUESTED** [review]

File: src/test/java/io/kestra/plugin/airflow/dags/TriggerDagRunTest.java
Comment (iNikitaGricenko) [reply src/test/java/io/kestra/plugin/airflow/dags/TriggerDagRunTest.java:1]: I have tested using docker-compose. But it's requires a lot of services. I have asked Anna if should i use docker-compose or just disable unit tests, so we decided to just disable them

File: src/main/java/io/kestra/plugin/airflow/dags/TriggerDagRun.java (hunk 3 above)
Comment (loicmathieu) [reply src/main/java/io/kestra/plugin/airflow/dags/TriggerDagRun.java:105]: I didn't see the description from Anna.

You can keep the options but I'd rather not use CheckFrequency and include a basic polling as we do in other plugins with only a maxDuration and a pollFrequency, you can take as example: https://github.com/kestra-io/plugin-airbyte/blob/master/src/main/java/io/kestra/plugin/airbyte/connections/CheckStatus.java

**Pull request merged**